applications/FluidDynamicsApplication/custom_processes/shock_capturing_physics_based_process.h

```diff
@@ -313,26 +300,21 @@ class KRATOS_API(FLUID_DYNAMICS_APPLICATION) ShockCapturingPhysicsBasedProcess :
             const double s_w = div_v_pow / (div_v_pow + rot_v_norm_pow + eps);
 
             // Calculate limited shock sensor
-            const double s_beta_0 = 0.01;
+            const double s_beta_min = 0.0;
             const double s_beta_max = 2.0 / std::sqrt(std::pow(gamma, 2) - 1.0);
+            const double s_beta_th = 0.01;
             const double s_beta = s_omega * s_w;
-            // const double s_beta_hat = LimitingFunction(s_beta, s_beta_0, s_beta_max);
-            const double s_beta_hat = SmoothedLimitingFunction(s_beta, s_beta_0, s_beta_max);
+            const double s_beta_hat = SmoothedLimitingFunction(s_beta,s_beta_min,s_beta_max);
             rElement.GetValue(SHOCK_SENSOR) = s_beta_hat;
 
             // Calculate elemental artificial bulk viscosity
-            const double k_beta = 1.5;
-            const double elem_b_star = (k_beta * h_beta / k) * ref_mom_norm * s_beta_hat;
+            const double elem_b_star = (mK_beta_constant * h_beta / k) * ref_mom_norm * s_beta_hat;
             rElement.GetValue(ARTIFICIAL_BULK_VISCOSITY) = elem_b_star;
 
             // Calculate elemental artificial conductivity (dilatancy)
             // Note that this is only required if the formulation is thermally coupled
             if (mThermallyCoupledFormulation) {
-                const double Pr_beta_min = 0.9;
-                const double alpha_pr_beta = 2.0;
-                const double mach_threshold = 3.0;
-                const double Pr_beta = Pr_beta_min * (1.0 + std::exp(-2.0 * alpha_pr_beta * (mach - mach_threshold)));
-                const double elem_k1_star = (gamma * c_v / Pr_beta) * elem_b_star;
+                const double elem_k1_star = (gamma * c_v / mPr_beta_constant) * elem_b_star;
```

Comment:
In Fernández et al. (2018) it is stated that both a the previous expression and the far-field Prandtl number can be used. I'd keep both options, in the following manner. We default the far-field Prandtl number to 0.0 and if the user provides it we overwrite this value. Then we check, if the value is non-zero we use that, if not we use my previous implementation. Makes sense for you?